src/tasks/component-installers/devfile-workspace-operator-installer.ts

```diff
@@ -59,64 +91,120 @@ export class DevWorkspaceTasks {
+            task.title = `${task.title}...It already exists.`
+          } else {
+            const clusterRolePath = path.join(this.getTemplatePath(ctx), 'devworkspace-controller-role.ClusterRole.yaml')
+            await this.kubeHelper.createClusterRoleFromFile(clusterRolePath, this.devworkspaceClusterRole)
+            task.title = `${task.title}...done.`
+          }
+        }
+      },
       {
         title: `Create ClusterRole ${this.devWorkspaceViewWorkspaceClusterRole}`,
-        task: async (_ctx: any, task: any) => {
+        task: async (ctx: any, task: any) => {
           const exist = await this.kubeHelper.clusterRoleExist(this.devWorkspaceViewWorkspaceClusterRole)
           if (exist) {
             task.title = `${task.title}...It already exists.`
           } else {
-            const clusterRolePath = path.join(this.getTemplatePath(), 'view-workspaces-cluster-role.yaml')
+            const clusterRolePath = path.join(this.getTemplatePath(ctx), 'devworkspace-controller-view-workspaces.ClusterRole.yaml')
             await this.kubeHelper.createClusterRoleFromFile(clusterRolePath, this.devWorkspaceViewWorkspaceClusterRole)
             task.title = `${task.title}...done.`
           }
         }
       },
+      {
+        title: `Create RoleBinding ${this.devWorkspaceLeaderElectionRoleBinding} in namespace ${this.getNamespace()}`,
+        task: async (ctx: any, task: any) => {
+          const exist = await this.kubeHelper.roleBindingExist(this.devWorkspaceLeaderElectionRoleBinding, this.getNamespace())
+          if (exist) {
+            task.title = `${task.title}...It already exists.`
+          } else {
+            const roleBindingPath = path.join(this.getTemplatePath(ctx), 'devworkspace-controller-leader-election-rolebinding.RoleBinding.yaml')
+            await this.kubeHelper.createRoleBindingFromFile(roleBindingPath, this.getNamespace())
+            task.title = `${task.title}...done.`
+          }
+        }
+      },
+      {
+        title: `Create ClusterRoleBinding ${this.devworkspaceProxyClusterRoleBinding}`,
+        task: async (ctx: any, task: any) => {
+          const exist = await this.kubeHelper.clusterRoleBindingExist(this.devworkspaceProxyClusterRoleBinding)
+          if (exist) {
+            task.title = `${task.title}...It already exists.`
+          } else {
+            const clusterRoleBindingPath = path.join(this.getTemplatePath(ctx), 'devworkspace-controller-proxy-rolebinding.ClusterRoleBinding.yaml')
+            const rawYaml = await fs.readFile(clusterRoleBindingPath, 'utf-8')
+            const clusterRoleBindingYaml: any = yaml.safeLoad(rawYaml)
+            clusterRoleBindingYaml.subjects[0].namespace = this.getNamespace()
+
+            await this.kubeHelper.createClusterRoleBindingFrom(clusterRoleBindingYaml)
+            task.title = `${task.title}...done.`
+          }
+        }
+      },
       {
         title: `Create ClusterRoleBinding ${this.devWorkspaceRoleBinding}`,
```

Comment:
```suggestion
        title: `Create ClusterRoleBinding ${this.devWorkspaceRoleBinding} in namespace ${this.getNamespace()}`,
```

Reply:
It is not namespaced object, we should not print ns

Reply:
Clusterrolebinding can be used for one namespace. And in the task code you can find such usage.

Reply:
My mistake